tests/seta/test_update_job_priority.py

```diff
@@ -0,0 +1,123 @@
+import datetime
+
+import pytest
+from mock import patch
+
+from treeherder.seta.models import JobPriority
+from treeherder.seta.runnable_jobs import RunnableJobsClient
+from treeherder.seta.update_job_priority import (_initialize_values,
+                                                 _sanitize_data,
+                                                 _unique_key,
+                                                 _update_table,
+                                                 query_sanitized_data)
+
+
+# Prevent requests to be used in any of the tests
+@pytest.fixture(autouse=True)
+def no_requests(monkeypatch):
+    monkeypatch.delattr("requests.sessions.Session.request")
```

Comment:
The only reason I was using this was to make sure we did not make external calls unintentionally. I can drop it.